src/components/edit/thematic/NoDataColor.js

```diff
@@ -14,7 +14,7 @@ const NoDataColor = ({ value, onChange }) => {
     return (
         <div>
             <Checkbox
-                label={i18n.t('Show no data')}
+                label={i18n.t('Include org. units with no data')}
```

Comment:
Remove the period after "org" for consistency with all other uses of "org unit" in the app.

Reply:
Fixed in [6a30ebb](https://github.com/dhis2/maps-app/pull/3204/commits/6a30ebbae978448c0ffd3cdacb5cfca722aa9ac7)